packages/ai/src/ai/common/models/audio/wav_to_mp3.py

```
@@ -0,0 +1,50 @@
+    import lameenc
+
+    with wave.open(wav_path, 'rb') as wf:
+        nchannels = wf.getnchannels()
+        sampwidth = wf.getsampwidth()
+        framerate = wf.getframerate()
+        frames = wf.readframes(wf.getnframes())
+
+    if sampwidth != 2:
+        raise ValueError(f'lameenc path requires 16-bit WAV, got sample width {sampwidth}')
+    if nchannels not in (1, 2):
+        raise ValueError(f'lameenc path supports 1–2 channels, got {nchannels}')
```

Ruff (0.15.7)

[warning] Avoid specifying long messages outside the exception class

(TRY003)